metrics/gleu/gleu.py

```diff
@@ -78,8 +78,10 @@ def get_gleu_stats(scores) :
             'ci': ci}
 
 class Gleu(nlp.Metric):
+    def __init__(self, **kwargs):
+        raise NotImplementedError("Gleu is currently under construction.")
```

Comment:
Hi. Are there any plans to include GLEU into this library? Are there any other codebases that you may recommend?